operator/v1/types_clustermanager.go

```diff
@@ -125,16 +125,36 @@ type RegistrationDriverHub struct {
 	// +kubebuilder:validation:Enum=csr;awsirsa
 	AuthType string `json:"authType,omitempty"`
 
+	// CSR represents the configuration for csr driver.
+	// +optional
+	CSR *CSRDriverConfig `json:"csr,omitempty"`
+
+	// AwsIrsa represents the configuration for awsisra driver.
+	// +optional
+	AwsIrsa *AwsIrsaConfig `json:"awsisra,omitempty"`
```

Comment:
```suggestion
	AwsIrsa *AwsIrsaDriverConfig `json:"awsisra,omitempty"`
```

To match the pattern of above `CSRDriverConfig`. I think both should either include "Driver" or both should omit it. Same comment for the other occurrences.